Algorithmization/src/com/company/task_2/Task_1.java

```diff
@@ -0,0 +1,36 @@
+package com.company.task_2;
+
+import java.util.Arrays;
+
+public class Task_1 {
+    public static void showArray(int[][] array) {
```

Comment:
move this functions (showArray,randomArray) to separate class (e.g. MatrixUtils). this class is an example of concept named Utility class - class which contains only static members and can't be instantiated (use private constructor)
then use this methods in all of the methods below